src/Main.purs

```diff
@@ -29,8 +34,8 @@ renderers :: Array Doc.Renderer
 renderers = Renderers.all
 
 -- json is a Foreign object whose type is defined in /cli/src/Main.d.ts
-main :: Json -> Either Error SourceCode
-main json = do
+main :: StrMap String -> Json -> Either Error SourceCode
+main optionStrings json = do
```

Comment:
This breaks `quicktype.io` so please bump the major release number in `package.json`.

Reply:
This is breaking my spirit already.

Reply:
Actually, I'll make a separate `mainWithOptions` function, so the API doesn't change at all.  And it's better if it's curried, obviously ;-)